libs/commons/Logs_.ml

```diff
@@ -267,27 +243,29 @@ let read_level_from_env vars =
 (* Enable basic logging (level = Logs.Warning) so that you can use Logging
  * calls even before a precise call to setup_logging.
  *)
-let enable_logging () =
+let setup_basic () =
   Logs.set_level ~all:true (Some Logs.Warning);
   Logs.set_reporter
-    (reporter ~dst:UFormat.err_formatter ~require_one_of_these_tags:[]
+    (mk_reporter ~dst:UFormat.err_formatter ~require_one_of_these_tags:[]
        ~read_tags_from_env_vars:[] ());
   ()
 
-let setup_logging ?(highlight_setting = Std_msg.get_highlight_setting ())
-    ?log_to_file:opt_file ?(skip_libs = default_skip_libs)
-    ?(require_one_of_these_tags = default_tags)
-    ?(read_level_from_env_vars =
-      [ "PYTEST_SEMGREP_LOG_LEVEL"; "SEMGREP_LOG_LEVEL" ])
-    ?(read_tags_from_env_vars =
-      [ "PYTEST_SEMGREP_LOG_TAGS"; "SEMGREP_LOG_TAGS" ]) ~level () =
+let setup ?(highlight_setting = Std_msg.get_highlight_setting ())
+    ?log_to_file:opt_file ?(require_one_of_these_tags = default_tags)
+    ?(read_level_from_env_vars = [ "LOG_LEVEL" ])
```

Comment:
Note GitLab was reporting perf problems due to all the logging that was produced by Semgrep, IIRC due to GitLab CI running with `LOG_LEVEL=debug`, Martin quick-fixed this by renaming it to `SEMGREP_LOG_LEVEL`. There are also changelog entries about these renames, so we would need to write more changelog entries for this.

Reply:
Logs_.ml is supposed to be independent of semgrep, but we can use SEMGREP_LOG_LEVEL in
the caller to Logs_.setup(), in CLI.ml and Core_CLI.ml

Reply:
There at least 3 levels with the same name:
* Semgrep, the organization or software vendor
* the `semgrep` repo (+ `semgrep-proprietary`) = libraries provided by the Semgrep vendor + Semgrep application
* the Semgrep application

The `SEMGREP_` prefix refers to the vendor, not the application.

Reply:
I think LOG_LEVEL is fine as a default, but yes I'll update the caller to not use this variable name.